homework/schedule/schedule.hpp

```diff
@@ -0,0 +1,43 @@
+#pragma once
+#include <chrono>
+#include <functional>
+#include <string>
+
+void schedule(std::function<void()> invokeFun, std::chrono::seconds duration) {
+    auto now = std::chrono::system_clock::now();
+    auto end = std::chrono::system_clock::now();
+
+    while (true) {
+        if (end >= (now + duration)) {
+            break;
+        }
+        end = std::chrono::system_clock::now();
+    }
```

Comment:
Use std::this_thread::sleeep for or sleep until instead loop. Do this in rest of functions